static/js/compose.js

```diff
@@ -283,6 +288,12 @@ export function finish() {
         return undefined;
     }
 
+    $("#compose-send-button").prop("disabled", true).trigger("blur");
```

Comment:
What code is supposed to re-enable the send button?

It's not very clear from reading the code.